src/components/parkingFineSummary/ParkingFineSummary.tsx

```diff
@@ -0,0 +1,150 @@
+            readOnly
+          />
+        </Card>
+
+        <TextInput
+          className="info-field"
+          id="fineTimeStamp"
+          label={t('common:fine-info:fine-timestamp:label')}
+          value={t('common:fine-info:fine-timestamp:placeholder')}
+          readOnly
+        />
+        <TextInput
+          className="info-field"
+          id="ref-number"
+          label={t('common:fine-info:ref-number:label')}
+          value={t('common:fine-info:ref-number:placeholder')}
+          readOnly
+        />
+
+        <TextInput
+          className="info-field"
+          id="address"
+          label={t('common:fine-info:address:label')}
+          value={t('common:fine-info:address:placeholder')}
+          readOnly
+        />
+        <TextInput
+          className="info-field"
+          id="additionalDetails"
+          label={t('common:fine-info:additional-details:label')}
+          value={t('common:fine-info:additional-details:placeholder')}
+          readOnly
+        />
+
+        <TextArea
+          className="info-field"
+          id="fine-title"
+          label={t('common:fine-info:fine-title:label')}
+          value={t('common:fine-info:fine-title:placeholder')}
+          readOnly
+        />
+        <TextInput
+          className="info-field"
+          id="additionalDetails"
+          label={t('common:fine-info:fine-details:label')}
+          value={t('common:fine-info:fine-details:placeholder')}
+          readOnly
+        />
+        <TextArea
+          className="info-field"
+          id="fine-title"
+          label={t('common:fine-info:fine-title:label')}
+          value={t('common:fine-info:fine-title:placeholder')}
+          readOnly
+        />
+        <TextInput
+          className="info-field"
+          id="additionalDetails"
+          label={t('common:fine-info:fine-details:label')}
+          value={t('common:fine-info:fine-details:placeholder')}
+          readOnly
+        />
+
+        <TextArea
+          className="wide-field"
+          id="address"
+          label={t('common:fine-info:fine-additional-details:label')}
+          value={t('common:fine-info:fine-additional-details:placeholder')}
+          readOnly
+        />
+        <TextInput
+          id="sum"
+          label={t('common:fine-info:sum:label')}
+          value={t('common:fine-info:sum:placeholder')}
+          readOnly
+          className="info-field"
+        />
+        <TextInput
+          className="info-field"
+          id="dueDate"
+          label={t('common:fine-info:due-date:label')}
+          value={t('common:fine-info:due-date:placeholder')}
+          readOnly
+        />
+      </div>
+
+      <TextInput
+        id="fineBarcode"
+        readOnly
+        label={t('common:fine-info:barcode:label')}
+        value={t('common:fine-info:barcode:placeholder')}
+      />
+      <Button type="button" variant="secondary" onClick={() => null}>
+        <IconCopy />
```

Comment:
You could use the iconLeft option of Button, like here: [https://hds.hel.fi/components/buttons/code#supplementary](https://hds.hel.fi/components/buttons/code#supplementary)